weetweet.py

```diff
@@ -104,65 +104,65 @@
     home_timeline="home", clear_nicks="cnicks", clear_buffer="clear",
     create_stream="stream", restart_home_stream="re_home")
 desc_dict = dict(
-    user="<user> [<id> <count>|<id>|<count>], Request user timeline, " +
-    "if <id> is given it will get tweets older than <id>, " +
+    user="<user> [<id> <count>|<id>|<count>], Request user timeline, " + \
```

Comment:
There is a lot of superfluous backslashed added. I think that they add a lot of unnecessary clutter.
(This applies for the rest of the backslashes also where they are not needed)